src/WebJobs.Script/Config/IFunctionsHostingConfiguration.cs

```diff
@@ -3,8 +3,10 @@
 
 namespace Microsoft.Azure.WebJobs.Script.Config
 {
-    internal interface IFunctionsHostingConfiguration
+    public interface IFunctionsHostingConfiguration
```

Comment:
why the change here?

Reply:
`FunctionsScaleManager`,`FunctionsScaleMonitorService`, `Utility` are public classes.